Jg/edit duke

jlgleason opened the pull request.

azunre commented:
1. https://github.com/NewKnowledge/duke-d3m-wrapper/blob/jg/edit_Duke/DukeD3MWrapper/wrapper.py#L9 this isn't used anywhere (import sys)

2. https://github.com/NewKnowledge/duke-d3m-wrapper/blob/jg/edit_Duke/DukeD3MWrapper/wrapper.py#L176 this still works if you don't set the parameter (in which case it will take the default value), correct?

3. https://github.com/NewKnowledge/duke-d3m-wrapper/blob/jg/edit_Duke/DukeD3MWrapper/wrapper.py#L126 This should be frame.value.columns etc Please test with provided python pipeline

azunre commented:
> 1. Not sure why that was in there - must have been from testing something! It's removed now.
> 2. Yes! I added a few lines so now it checks if the hyperparameter dictionary is empty, and if it is, it adds the 'records' hyperparameter with the default value (1)
> 3. Testing works with the provided python pipeline on the docker image! Also, frame.value is now only needed in one line (122) - the line that sub-samples from the dataset!

Lines 22 and 23 of wrapper.py are redundant I think, then it should be ready for merge

jlgleason commented:
> > 1. Not sure why that was in there - must have been from testing something! It's removed now.
> > 2. Yes! I added a few lines so now it checks if the hyperparameter dictionary is empty, and if it is, it adds the 'records' hyperparameter with the default value (1)
> > 3. Testing works with the provided python pipeline on the docker image! Also, frame.value is now only needed in one line (122) - the line that sub-samples from the dataset!
> 
> Lines 22 and 23 of wrapper.py are redundant I think, then it should be ready for merge

Deleted Line 22!
